bin/runtime/src/lib.rs

```diff
@@ -699,6 +701,7 @@ parameter_types! {
     // Maximum size of the lazy deletion queue of terminated contracts.
     pub const DeletionQueueDepth: u32 = 128;
     pub Schedule: pallet_contracts::Schedule<Runtime> = Default::default();
+    pub CodeHashLockupDepositPercent: Perbill = Perbill::from_percent(30);
```

Comment:
Where is the value 30% coming from?

Reply:
the docs for `CodeHashLockupDepositPercent`
```
/// The percentage of the storage deposit that should be held for using a code hash.
/// Instantiating a contract, or calling `add_delegate_dependency` (for contract delegation) protects
/// the code from being removed. In order to prevent abuse these actions are protected with
/// a percentage of the code deposit.
```
The 30% is the one used in substrate node runtime. I think we want to think about what value we would like here.

Reply:
Resolved off github